src/app/work-item/work-item-list/work-item-list.component.ts

```diff
@@ -175,14 +175,39 @@ export class WorkItemListComponent implements OnInit, AfterViewInit {
   }
 
   // event handlers
+  onToggle(entryComponent: WorkItemListEntryComponent, event: MouseEvent): void {
+    // This condition is to select a single work item for movement
+    // de-check, de-select, and close detail's panel
+    //for prior selected element (if any)
+    if (this.selectedWorkItemEntryComponent && this.selectedWorkItemEntryComponent != entryComponent){
+      this.selectedWorkItemEntryComponent.deselect();
+      this.selectedWorkItemEntryComponent.decheck();
+      this.panelState = 'out';
```

Comment:
Does changing the panelState to out actually change the URL from details page to list page?